src/main/resources/application.yaml

```diff
@@ -102,7 +102,7 @@ integrations:
     fiona:
         url: http://fiona-aurora.${openshift.cluster}.paas.skead.no
     herkimer:
-        url: false
+        url: https://herkimer-aurora.${openshift.cluster}.paas.skead.no/
```

Comment:
Burde vi la boober gå mot herkimer? Jeg har tenkt tidligere at det er uheldig med tanke på at den herkimer er i vår "prod". Eventuelt burde vi hatt en herkimer i aurora-dev som alltid kjører?

Reply:
Har det noen konsekvenser at å gå mot "prod" instansen?

Reply:
Ja og nei. Siden boober lager ressurser og credentials i herkimer tenker jeg heller på risikoen hvis en som utvikler gjør feil. Det hadde vært kjipt hvis noen mister noe data der. Men kanskje jeg som overtenker det og at det er et ikke problem

Reply:
Skjønner, men vi kan godt sette opp en ekstra bare for å være på den sikre siden.

Reply:
Hva tenker du @bentsolheim ? Er det bare jeg som er paranoid? :P

Reply:
Nei, jeg er enig. Under dev bør vi ikke gå mot prod-instansen av herkimer. Vi kan heller legge opp til å synce prod-db til dev-herkimer med et script ved behov.